### PR TITLE
fix: adding back failing test

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
I believe it was failing because GitHub didn't have memory enough to instantiate three containers. This PR removes one and refactors a bit.

### CODE REVIEW

File: sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegrationTest.java [hunk 1]
```diff
@@ -37,20 +38,25 @@ public TracingIntegrationTest(ApplicationContext applicationContext) {
         super(applicationContext, config);
     }
 
-    @Disabled //disabled ATM for https://github.com/lightbend/kalix-jvm-sdk/pull/1810
+    @Test //disabled ATM for https://github.com/lightbend/kalix-jvm-sdk/pull/1810
```
Comment (johanandren) [inline sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegrationTest.java:41]: No longer disabled?

Comment (franciscolopezsancho) [reply sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegrationTest.java:41]: That's what I was trying to make to work. Sorry, I'll delete the comment. It's confusing, ofc.

Comment (franciscolopezsancho) [reply sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegrationTest.java:41]: The whole idea is to enable the test

**Comment by franciscolopezsancho** [comment]
I'll come back to this later in the future. For now I'm closing it to remove the noise.

**Pull request closed without merging**